README.md

```diff
@@ -178,6 +178,7 @@ A collection of awesome browser-side [JavaScript](https://developer.mozilla.org/
 * [meteor](https://github.com/meteor/meteor) - An ultra-simple, database-everywhere, data-on-the-wire, pure-javascript web framework.
 * [ractive](https://github.com/ractivejs/ractive) - Next-generation DOM manipulation.
 * [vue](https://github.com/vuejs/vue) - Intuitive, fast & composable MVVM for building interactive interfaces.
+* [svelte](https://github.com/sveltejs/svelte) - Svelte is a tool for building fast web applications. It is similar to JavaScript frameworks such as React and Vue, which share a goal of making it easy to build slick interactive user interfaces.
```

Comment:
How about this ??

> "Svelte is a new way to build web applications. It's a compiler that takes your declarative components and converts them into efficient JavaScript that surgically updates the DOM."

Reply:
This is their own description: https://github.com/sveltejs/svelte#what-is-svelte
I'd go for this, indeed. Thanks!

Reply:
ok.. i appreciate that... good job !!!